src/tribler-core/tribler_core/session.py

```diff
@@ -254,21 +231,22 @@ def get_ipv8_statistics(self):
         """Return a dictionary with IPv8 statistics."""
         return TriblerStatistics(self).get_ipv8_statistics()
 
+    @inject
```

Comment:
I believe this is not necessary here, since there are not `Provide` statements in the method's arguments. Correct me if I'm wrong.

Reply:
You are right! It is just an artifact.